### ISSUE TITLE
does z.lua have word mode completion like that in fasd?

### ISSUE THREAD

**Issue by ssh352** [skywind3000/z.lua#102]
> Word mode completion can be triggered on any command. Word completion is triggered by any command line argument that starts with , (all), f, (files), or d, (directories), or that ends with ,, (all), ,,f (files), or ,,d (directories). Examples:

Thanks

**Comment by skywind3000** [comment]
no, tracking directory changing is enough, tracking every file name in the command arguments is a performance killer.
